index.html

```diff
@@ -195,19 +197,19 @@ <h1 class="text-center">Sharpen Your Skills</h1>
     <br><br>
     <table class="center">
       <tr>
-        <th><i class="fas fa-atom"></i>&nbsp; <a href="#">Data Structures</a></th>
-        <th><i class="fas fa-project-diagram"></i>&nbsp; <a href="#">Algorithms</a></th>
-        <th><i class="fab fa-python"></i>&nbsp; <a href="#">Python</a></th>
+        <th><i class="fas fa-atom"></i>&nbsp; <a href="https://www.geeksforgeeks.org/data-structures/" target="_blank">Data Structures</a></th>
+        <th><i class="fas fa-project-diagram"></i>&nbsp; <a href="https://leetcode.com/problemset/algorithms/" target="_blank">Algorithms</a></th>
+        <th><i class="fab fa-python"></i>&nbsp; <a href="https://docs.python.org/3/tutorial/index.html" target="_blank">Python</a></th>
       </tr>
       <tr>
-         <th><i class="fab fa-cuttlefish"></i>&nbsp; <a href="#">C</a></th>
-        <th><i class="devicon-cplusplus-line-wordmark fa-lg"></i>&nbsp; <a href="#">C++</a></th>
-        <th><i class="fab fa-java fa-lg"></i>&nbsp; <a href="#">Java</a></th>
+         <th><i class="fab fa-cuttlefish"></i>&nbsp; <a href="https://www.youtube.com/watch?v=KJgsSFOSQv0" target="_blank">C</a></th>
+        <th><i class="devicon-cplusplus-line-wordmark fa-lg"></i>&nbsp; <a href="https://www.cplusplus.com/" target="_blank">C++</a></th>
+        <th><i class="fab fa-java fa-lg"></i>&nbsp; <a href="https://www.youtube.com/watch?v=eTXd89t8ngI&list=PLd3UqWTnYXOmx_J1774ukG_rvrpyWczm0" target="_blank">Java</a></th>
       </tr>
       <tr>
-        <th><i class="fas fa-gem"></i>&nbsp; <a href="#">Ruby</a></th>
-        <th><i class="fas fa-square-root-alt"></i>&nbsp; <a href="#">Mathematics</a></th>
-        <th><i class="fa fa-database"></i>&nbsp; <a href="#">Databases</a></th>
+        <th><i class="devicon-cplusplus-line-wordmark fa-lg"></i>&nbsp; <a href="https://www.youtube.com/watch?v=g-1Cn3ccwXY" target="_blank">STL</a></th>
```

Comment:
STL is a part of CPP only so revert it back to Ruby only if you are unable to find any correct resources you can ask us or leave it blank.